Configuration/StandardSequences/python/RawToDigi_cff.py

```diff
@@ -111,6 +111,7 @@
 
 from Configuration.Eras.Modifier_run2_GEM_2017_cff import run2_GEM_2017
 run2_GEM_2017.toReplaceWith(RawToDigi, _gem_RawToDigi)
+run2_GEM_2017.toModify(muonGEMDigis, useDBEMap = cms.bool(True))
```

Comment:
This customization would be better done in the file where `muonGEMDigis` is defined.

Reply:
similarly

Reply:
Same comment as https://github.com/cms-sw/cmssw/pull/22619#discussion_r175019236.